.github/workflows/sbom.yml

```diff
@@ -0,0 +1,71 @@
+on:
+  release:
+    types: [published]
```

Comment:
published or tagged - I believe we would like to have SBOM published w/ every release.